Documentation November quick wins (#261)
opendatasoft/ods-documentation: source/publishing_data/01_creating_a_dataset/creating_dataset_with_multiple_files.rst

.. admonition:: Important
   :class: important

>    When synchronizing from a remote FTP location, Opendatasoft keeps a persistent cache and does not automatically prune files missing from the remote directory. Please use the "Clean cache" button on the ressource if you need some cleanup to be performed.

Reviewer: - "ressource" is a French word, you should replace it with "resource".
-  It's great to be polite but using "please" in instructions is overdoing the politeness.
- For the second sentence, it's better to use the conditional clause first. The reader can skip the sentence if the condition doesn't apply.
```suggestion
   When synchronizing from a remote FTP location, Opendatasoft keeps a persistent cache and does not automatically prune files missing from the remote directory. If you need some cleanup to be performed, to the right of the resource, click "Clean cache".
```
